include/rocksdb/utilities/options_type.h

```diff
@@ -406,6 +408,99 @@ class OptionTypeInfo {
         });
   }
 
+  // Create a new std::shared_ptr<Customizable> OptionTypeInfo
+  // This function will call the T::CreateFromString method to create a new
+  // std::shared_ptr<T> object.
+  //
+  // @param offset The offset for the Customizable from the base pointer
+  // @param ovt How to verify this option
+  // @param flags, Extra flags specifying the behavior of this option
+  // @param _sfunc Optional function for serializing this option
+  // @param _efunc Optional function for comparing this option
+  template <typename T>
+  static OptionTypeInfo AsCustomS(int offset, OptionVerificationType ovt,
```

Comment:
The name ended with S, U, P do not help to understand the function. Can we directly use like AsCustomSharedPtr, AsCustomUniquPtr, AsCustomRawPtr?